tools/bin/acceptance_test.sh

```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Indeed. This area of the docs is in need of an overhaul, most likely ever since the platform repo was spun off. I assume it's OK if I do a pass in a separate PR, besides, I have a few other changes in flight for after this PR merges.